.github/workflows/osrm-backend.yml

```diff
@@ -20,6 +20,27 @@ env:
   ENABLE_NODE_BINDINGS: "ON"
 
 jobs:
+  windows:
```

Comment:
I forgot about this with the Docker job.

[ci-complete](https://github.com/Project-OSRM/osrm-backend/pull/6312/files#diff-4f680fa9a2e9b55e339a5fc727a44e968318e00f54beb55da0e5c5b24bbd2f9dR642) is the meta-job that PRs will block merging on. We should add `windows` and `docker` to the list of `needs`, in the unlikely case that all the Linux/Mac jobs succeed before those two complete/fail.

Reply:
Okay, will do. Btw while we are waiting for someone to disable Appveyor check, may be just return that job and make it no-op(i.e. always green?). It would unblock this PR from being merged and would allow to work on other PRs which depend on it.

Reply:
Good idea :+1: